src/components/Sidebar/Sidebar.vue

```diff
@@ -141,6 +146,15 @@ export default {
       }
     },
 
+    toggleSidebar() {
+      if (window.matchMedia) {
+        this.$emit(
+          'update:minimize',
+          window.matchMedia('(max-width: 1000px)').matches
```

Comment:
Good work, my friend.
As this is an open source project, it might be a good idea to add a new prop where the user can set the max-width to which they want the sidebar to automatically collapse.